src/blocklyc.js

```diff
@@ -647,7 +629,7 @@ function loadInto(modal_message, compile_command, load_option, load_action) {
 
                 if (client_version >= minCodedVer) {
                     //Request load with options from BlocklyProp Client
-                    $.post(client_url + 'load.action', {
+                    $.post("http://" + client_domain_name + ":" + client_domain_port + "/load.action", {
```

Comment:
eliminating the `client_url` variable by using it's components.  This gets a further fix in the next PR.

Reply:
client_domain_name and client_domain_port are defaulted to localhost:6009. This can be overridden from the UI.